lib/estella/helpers.rb

```diff
@@ -82,10 +92,6 @@ def refresh_index!
         __elasticsearch__.refresh_index!
       end
 
-      def set_index_alias!(name)
```

Comment:
I believe we only ever used this for migrations - we don't seem to use it anywhere any longer.